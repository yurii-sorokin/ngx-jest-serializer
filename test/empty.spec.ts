import { initHostFixture } from './framework/host'

describe(`Empty`, () => {
  it(`should print empty component`, () => {
    const fixture = initHostFixture({})

    expect(fixture).toMatchSnapshot()
  })
})
