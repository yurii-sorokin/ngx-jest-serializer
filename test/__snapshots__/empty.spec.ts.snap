// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty should print empty component 1`] = `<tst-host />`;
